classes/notification/form/NotificationSettingsForm.inc.php

```diff
@@ -22,13 +22,7 @@ class NotificationSettingsForm extends PKPNotificationSettingsForm {
 	 * @copydoc PKPNotificationSettingsForm::getNotificationSettingsMap()
 	 */
 	protected function getNotificationSettingsMap() {
-		$settingsMap = parent::getNotificationSettingsMap();
-		$settingsMap[NOTIFICATION_TYPE_PUBLISHED_ISSUE] = array(
-			'settingName' => 'notificationPublishedIssue',
-			'emailSettingName' => 'emailNotificationPublishedIssue',
-			'settingKey' => 'notification.type.issuePublished',
-		);
-		return $settingsMap;
+		return NotificationManager::getNotificationSettingsMap();
```

Comment:
Let's get rid of `NotificationSettingsForm::getNotificationSettingsMap()` and `PKPNotificationSettingsForm::getNotificationSettingsMap()` since they're not used now.